core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -725,7 +723,7 @@ private static String getAdditionalLogging(final PipeLineSession pipeLineSession
 	public void registerReceiver(Receiver<?> receiver) {
 		receivers.add(receiver);
 		// Cast arguments to String before invocation so that we do not have recursive call to logger when trace-level logging is enabled
-		if (log.isDebugEnabled()) log.debug("Adapter [{}] registered receiver [{}] with properties [{}]", name, receiver.getName(), receiver.toString());
+		if (log.isDebugEnabled()) log.debug("Adapter [{}] registered receiver [{}] with properties [{}]", name, receiver.getName(), receiver);
```

Comment:
Ah yes, good spotted. I've added a SuppresWarnings for this, so it is not found in the future.
The too much details issue I'd like to leave for now, because I otherwise need to find out what I do need etc.